meta-cube/recipes-support/overc-utils/source/cube-ctl

```diff
@@ -1011,9 +1011,9 @@ case "${cmd}" in
 	    out_dir="${container_dir}/"
 	fi
 
-	# if [ -n "${track_containers}" ]; then
-	#     ${SBINDIR}/overc-ctl track ${container_name} -o ${out_dir}
-	# fi
+	if [ -n "${track_containers}" ]; then
+	    ${SBINDIR}/overc-ctl track ${container_name} -o ${out_dir}
+	fi
```

Comment:
This is too soon to start tracking. That's why it was moved.

For containers like docker, etc, there is a two phase addition of the container. At
the bottom, all the movement is done and then we should track it.